spec/javascripts/spec_helper.coffee

```diff
@@ -1,7 +1,11 @@
 #= require support/bind-poly
 #= require application
 #= require angular-mocks/angular-mocks
-beforeEach ->
-  this.addMatchers
-    toEqualData: (expected)-> return angular.equals(this.actual, expected)
+#= require support/phantomjs-shims
 
+beforeEach ->
```

Comment:
Just for my own edification, this is the major change that fixes things, right?  We are using master of teaspoon which includes the change to how their matchers worked.